specs/BackgroundColor.md

```diff
@@ -0,0 +1,111 @@
+
+
+# API Details
+## Win32 C++
+```cpp
+// This is the ICoreWebView2Controller2 interface
+interface ICoreWebView2Controller2 : ICoreWebView2Controller {
+
+  /// The `BackgroundColor` property allows developers to set the color that
+  /// shows when WebView has not loaded any web content and when a webpage
+  /// does not specify a background color. Color is specified by the
+  /// COREWEBVIEW2_COLOR value meaning the background color can also be
+  /// transparent.
+  /// The WebView background color will show before the initial navigation,
+  /// between navigations before the next page has rendered, and for pages with
+  /// no `background` style properties set. To clarify the latter case, WebView
+  /// will always honor a webpage's background content. `BackgroundColor` will
+  /// only show in the absence of css `background` style properties. In that
+  /// case, WebView will render web content over the `BackgroundColor` color.
+  /// For a transparent background, web content will render over hosting app
+  /// content. WebView's default background color is white to match the browser
+  /// experience.
+  /// It is important to note that while COREWEBVIEW2_COLOR has `A` an alpha
+  /// value, semi-transparent colors are not supported by this API and setting
+  /// `BackgroundColor` to a semi-transparent color will fail with E_INVALIDARG.
+  /// Any alpha value above 0 and below 255 will result in an E_INVALIDARG
+  /// error.`BackgroundColor` can only be an opaque color or transparent.
+  ///
+  /// The `BackgroundColor` property enables a seamless UI experience.
+  /// Developers can replace the 'white flash' between loading pages with a
+  /// color better suited to their application. For websites with no specified
+  /// background color, developers can display web contents over a color of
+  /// their choosing. They can also do away with the background color entirely
+  /// with transparency and have the 'in between pages color' just be hosting
+  /// content, or have hosting app content be the backdrop for webpages without
+  /// a background color specified.
+  ///
+  /// \snippet ViewComponent.cpp BackgroundColor
+
+  [propget] HRESULT BackgroundColor([out, retval] COREWEBVIEW2_COLOR* backgroundColor);
+
+  /// Sets the `BackgroundColor` property.
+
+  [propput] HRESULT BackgroundColor([in] COREWEBVIEW2_COLOR backgroundColor);
+}
+
+
+/// A value representing RGBA color (Red, Green, Blue, Alpha) for WebView2.
+/// Each component takes a value from 0 to 255, with 0 being no intensity
+/// and 255 being the highest intensity.
+
+typedef struct COREWEBVIEW2_COLOR {
+
+  /// Specifies the intensity of the Alpha ie. opacity value. 0 is transparent,
+  /// 255 is opaque.
+
+  BYTE A;
+
+  /// Specifies the intensity of the Red color
+
+  BYTE R;
+
+  /// Specifies the intensity of the Green color
+
+  BYTE G;
+
+  /// Specifies the intensity of the Blue color
+
+  BYTE B;
+} COREWEBVIEW2_COLOR;
+```
+## .NET and WinRT
```

Comment:
You've listed just the WinRT IDL3. You can change this to just '## WinRT' since you don't have the .NET API definition.